operators/pkg/controller/elasticsearch/driver/default.go

```diff
@@ -327,7 +274,7 @@ func (d *defaultDriver) Reconcile(
 		)
 	}
 
-	res = d.reconcileNodeSpecs(es, podTemplateSpecBuilder, esClient, observedState)
+	res = d.reconcileNodeSpecs(es, esReachable, podTemplateSpecBuilder, esClient, observedState)
```

Comment:
Is there a particular reason to not check esReachable and requeue the reconcilliation before calling `reconcileNodeSpecs`?

Reply:
We still want to create additional pods we can create: this does not require the cluster to respond to requests.

2 main use cases:
1. There are 0 nodes in the cluster, we need to create the first ones :)
2. User requests cluster scale-up, we can do it even though ES does not respond.

Reply:
:ok_hand: :no_mouth: